Localization/en-US.toml

```diff
@@ -36,6 +36,9 @@ MaxBuffSlots.Tooltip = "Configures the maximum amount of buffs a player may use.
 BossBagSellPriceAveraging.Label = "Treasure Bag Sell Price Averaging"
 BossBagSellPriceAveraging.Tooltip = "If enabled, Treasure Bags will have their sell prices auto-generated from the average sell price of their contents.\nThe price may vary slightly when reloaded."
 
+RespawnTimeModifier.Label = "Respawn Time Modifier"
+RespawnTimeModifier.Tooltip = "A percent modifier to your respawn time.\nDoes not take effect during boss fights."
```

Comment:
I can't quite put my finger on it, but this phrasing feels slightly off. "[...] to your [...]" is probably the culprit, but I cannot think of any better ways to phrase it.

Reply:
How does "A percent modifier to the time it takes to respawn" sound?

Reply:
Might be better to show it as a multiplier and say "Multiplicatively modifies your respawn time."

Reply:
I can't change the way the number is displayed. Its hardcoded by tML.

I avoided making it a float option that functions how you suggested but floats aren't rounded or truncated at all when displayed. I can look into circumnavigating this with the increment attribute.

Reply:
```cs
private float field;
public float Property
{
		get => field;
		set => field = RoundingOrTruncatingMethod(value);
}
```